driver-async/src/main/com/mongodb/async/client/ListDatabasesIterable.java

```diff
@@ -43,4 +45,25 @@
      * @mongodb.driver.manual reference/method/cursor.batchSize/#cursor.batchSize Batch Size
      */
     ListDatabasesIterable<T> batchSize(int batchSize);
+
+    /**
+     * Sets the query filter to apply to the returned database names.
+     *
+     * @param filter the filter, which may be null.
+     * @return this
+     * @since 3.6
+     * @mongodb.server.release 3.6
```

Comment:
Added in MongoDB 3.4.2 according to server docs.  Did you make it 3.6 because it was added in a patch release?

Reply:
just following the spec but will update.